deps/v8/src/snapshot/embedded/platform-embedded-file-writer-base.h

```diff
@@ -5,6 +5,8 @@
 #ifndef V8_SNAPSHOT_EMBEDDED_PLATFORM_EMBEDDED_FILE_WRITER_BASE_H_
 #define V8_SNAPSHOT_EMBEDDED_PLATFORM_EMBEDDED_FILE_WRITER_BASE_H_
 
+#include "src/objects/fixed-array-inl.h"
```

Comment:
It's pretty weird you are including local header from a header. Is it needed to be included here or it coud be included in the .cc file?

Reply:
it looks like it makes sense to declare it here. Otherwise the issue should be fixed in 4 files which include this one